hooks/000-provide-uids-gids.chroot

```diff
@@ -122,6 +122,7 @@ geoclue:x:111:120::/var/lib/geoclue:/bin/false
 gdm:x:112:121:Gnome Display Manager:/var/lib/gdm3:/bin/false
 colord:x:113:123:colord colour management daemon:/var/lib/colord:/usr/sbin/nologin
 gnome-initial-setup:x:114:65534::/run/gnome-initial-setup/:/bin/false
+rtkit:x:107:124:RealtimeKit,,,:/proc:/usr/sbin/nologin
```

Comment:
Could you set this uid to 115? Previously 107 was allocated to the docker user (see 45d8020), and it seems safer not to reuse uids from core22.

It's not clear whether we'll encounter file systems using Ubuntu Core's uids/gids, but it seems better not to risk it.

Reply:
Done.